backend/tests/test_logging_utils.py

```
@@ -17,14 +22,72 @@ def test_log_warning_does_not_emit_sensitive_kwargs(caplog):
         )
 
     record = caplog.records[-1]
     assert record.getMessage() == "event=security_event"
-    assert not hasattr(record, "password")
+    assert not hasattr(record, _SENSITIVE_ATTR)
     assert not hasattr(record, "nested")
     assert not hasattr(record, "details")
-    assert not hasattr(record, "authorization")
+    assert not hasattr(record, _AUTH_CONTEXT_FIELD)
+
+import asyncio
```

Ruff (0.15.2)

[error] Module level import not at top of file

(E402)